README.md

```diff
@@ -1,87 +1,98 @@
-<p align="center"><img width=70% src="https://github.com/SwiftLeeds/swiftleeds-web/blob/main/media/swift-leeds-horizontal.png"></p>
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="./media/swift-leeds-horizontal-dark.png">
+        <img alt="The Swift Leeds logo showing two swift birds to the left of the text 'Swift Leeds'" src="./media/swift-leeds-horizontal.png">
+    </picture>
+</p>
 
 ![Contributions welcome](https://img.shields.io/badge/contributions-welcome-orange.svg)
 [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://opensource.org/licenses/MIT)
   
 ## Introduction 👋🏼
 
-SwiftLeeds is a brand new Swift conference that started in 2021. It's a truly unique conference that is built by the community for the community. Hosted in the heart of Leeds City, UK.
+SwiftLeeds is a truly unique conference built by the community for the community. Started in 2021, we're now working towards our third year. SwiftLeeds is hosted in the heart of [Leeds](https://www.google.com/maps/place/Leeds/@53.8059821,-1.6057715,12z/data=!3m1!4b1!4m5!3m4!1s0x48793e4ada64bd99:0x51adbafd0213dca9!8m2!3d53.8007554!4d-1.5490774), a beautiful city in the northern English county of Yorkshire.
 
-The website is split into two components: the API and the front-end. Our immediate focus is developing a solid API that can power multiple platforms. Over the next few weeks, we'll be adding more and more front-end content to complement our API efforts.
+The website, which is of course built in Swift using [Vapor](https://vapor.codes/), is split into two key components:
```

Comment:
I'd say there's 3 components.  
- Front-end web used by customers
- Admin area to add/update detils
- API's for the App